releasetasks/templates/desktop/binary_transparency.yml.tmpl

```diff
@@ -0,0 +1,38 @@
+{% set buildername = "release-{}-{}_binary_transparency".format(branch, product) %}
+-
+    taskId: "{{ stableSlugId(buildername) }}"
+    requires:
+        - "{{ stableSlugId('release-foo-{}_chcksms'.format(product)) }}"
+    reruns: 5
+    task:
+        provisionerId: "scriptworker-prov-v1"
+        workerType: "dummy-worker-transpar"
```

Comment:
Yup, the dummy workers are for now and will be addressed as binary transparency is rolled out.